README.md

```diff
@@ -0,0 +1,754 @@
+
+# [UCAN Envelope] Configuration
+
+## Type Tag
+
+The UCAN envelope tag for UCAN Delegation MUST be set to `ucan/dlg@1.0.0-rc.1`.
+ 
+## Delegation Payload
+
+The Delegation payload MUST describe the authorization claims, who is involved, and its validity period.
+
+| Field   | Type                                      | Required | Description                                                 |
+|---------|-------------------------------------------|----------|-------------------------------------------------------------|
+| `iss`   | `DID`                                     | Yes      | Issuer DID (sender)                                         |
+| `aud`   | `DID`                                     | Yes      | Audience DID (receiver)                                     |
+| `sub`   | `DID`                                     | Yes      | Principal that the chain is about (the [Subject])           |
+| `cmd`   | `String`                                  | Yes      | The [Command] to eventually invoke                          |
+| `pol`   | `Policy`                                  | Yes      | [Policy]                                                    |
+| `nonce` | `Bytes`                                   | Yes      | Nonce                                                       |
+| `meta`  | `{String : Any}`                          | No       | [Meta] (asserted, signed data) — is not delegated authority |
+| `nbf`   | `Integer` (53-bits[^js-num-size])         | No       | "Not before" UTC Unix Timestamp in seconds (valid from)     |
+| `exp`   | `Integer \| null` (53-bits[^js-num-size]) | Yes      | Expiration UTC Unix Timestamp in seconds (valid until)      |
+
+# Capability
+
+Capabilities are the semantically-relevant claims of a delegation. They MUST be presented as a map under the `cap` field as a map. This map is REQUIRED but MAY be empty. This MUST take the following form:
+
+| Field  | Type      | Required | Description                                                                                      |
+|--------|-----------|----------|--------------------------------------------------------------------------------------------------|
+| `sub`  | `DID`     | Yes      | The [Subject] that this Capability is about                                                      |
+| `cmd`  | `Command` | Yes      | The [Command] of this Capability                                                                 |
+| `pol`  | `Policy`  | Yes      | Additional constraints on eventual Invocation arguments, expressed in the [UCAN Policy Language] |
+
+Here is an illustrative example:
+
+``` js
+{
+  // ...
+  "sub": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp"
+  "cmd": "/blog/post/create",
+  "pol": [
+    ["==", ".status", "draft"],
+    ["every", ".reviewer", ["match", ".email", "*@example.com"]],
```

Comment:
We had gone back and forth on `like` and `match` several times. The argument against `like` is that we don't support single character matching, so it could be confusing. But also, I bet that very few people will make the connection to SQL, and I don't really have strong opinions. I had `match` in the Rust code from way back when so went with that, but can switch to `like` since it sounds like you do care about it 👍

Reply:
This needed changes in a bunch of places (hence not just merging the `suggestion`), but its' updated now ✅